docs/api-docs/store-logs/store-logs.md

```diff
@@ -0,0 +1,329 @@
+      "date_created": "2021-12-22T23:44:15+00:00"
+    }
+  ],
+  "meta": {
+    "pagination": {
+      "total": 2,
+      "count": 2,
+      "per_page": 50,
+      "current_page": 1,
+      "total_pages": 1,
+      "links": {
+        "current": "?id%3Ain=5%2C7&page=1&limit=50"
+      }
+    }
+  }
+}
+```
+
+<!-- type: tab-end -->
+
+
+### Filter by type 
+
+To filter the entries by type, use the `type` query parameter. Examples of log type include `general`, `payment`, and `shipping`. For a list of log types, consult the [API endpoint reference](/api-reference/store-management/store-logs/system-logs/get-system-logs).  The following example requests `design`-type entries: 
+
+<!--
+type: tab
+title: Request
+-->
+
+```http title="Example request: Filter by type" lineNumbers
+GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/store/systemlogs?type=design
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+```
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example response: Filter by type" lineNumbers
+{
+  "data": [
+    {
+      "id": 1,
+      "type": "design",
+      "module": "Theme Download",
+      "severity": "success",
+      "summary": "user@example.com Downloaded Cornerstone-6.2.0.zip, 2zs05d70-3fb3-113m-43t0-179242c3acd9",
+      "message": "user@example.com Downloaded Cornerstone-6.2.0.zip, 2zs05d70-3fb3-113m-43t0-179242c3acd9",
+      "date_created": "2021-12-16T19:10:26+00:00"
+    },
+    {
+      "id": 6,
+      "type": "design",
+      "module": "Theme Download",
+      "severity": "success",
+      "summary": "user@example.com Downloaded LifeStyle-1.4.1.zip, 8e2a49c1-f85d-3149-ghce-5e2ba6d55p34",
+      "message": "user@example.com Downloaded LifeStyle-1.4.1.zip, 8e2a49c1-f85d-3149-ghce-5e2ba6d55p34",
+      "date_created": "2021-12-22T15:27:09+00:00"
+    }
+  ],
+  "meta": {
+    "pagination": {
+      "total": 2,
+      "count": 2,
+      "per_page": 50,
+      "current_page": 1,
+      "total_pages": 1,
+      "links": {
+        "current": "?type=design&page=1&limit=50"
+      }
+    }
+  }
+}
+```
+
+<!-- type: tab-end -->
+
+
+### Filter by module 
+
+To filter the entries by `module`, use the `module` query parameter. Modules further categorize the type of log. For example, the log type `shipping` has many modules including `Zone based free shipping (freeshipping)` and `FedEx Ground Home Delivery (5-7 Business Days) (flatrate)`. For a list of log modules, consult the [API endpoint reference](/api-reference/store-management/store-logs/system-logs/get-system-logs). The following example requests entries from the `order status` module: 
```

Comment:
Need to get more explanation of what a module is

Reply:
I'm wondering if we should have a separate section of all the available modules as something like an appendix (outside of the Get system logs endpoint?

Reply:
I actually asked around, and I don't believe I can get my hands on a list of modules. I re-organized the doc so that modules centers around a specific use case (rather than needing all the modules)